app/components/learn-showcase.js

```diff
@@ -0,0 +1,68 @@
+import Component from '@ember/component';
+
+const showcase = [
```

Comment:
The model in the `learn/examples` route already contains this data and it shouldn't be defined in 2 places: https://github.com/ember-learn/ember-website/blob/master/app/routes/learn.js#L5